mountpoint-s3-fs/src/superblock.rs

```diff
@@ -1418,26 +1411,72 @@ impl LookedUp {
     }
 }
 
-/// A wrapper around a `HashMap<InodeNo, Inode>`` that just takes care of metrics when inodes are
-/// added or removed.
+/// A wrapper around a `HashMap<InodeNo, (Inode, u64)>`` that takes care of managing inodes and their
```

Comment:
typo:
```suggestion
/// A wrapper around a `HashMap<InodeNo, (Inode, u64)>` that takes care of managing inodes and their
```

Reply:
Also consider focusing on what it does ("maps InodeNo to Inode & lookup count") rather than the internal struct it wraps.